test/support/api/programs.js

```diff
@@ -1,37 +1,55 @@
 import _ from 'underscore';
 import { getResource, getRelationship, mergeJsonApi } from 'helpers/json-api';
 
-import fxPrograms from 'fixtures/test/programs';
+import fxPrograms from 'fixtures/collections/programs';
 
 import { getProgramFlows } from './program-flows';
 import { getProgramActions } from './program-actions';
-import { getWorkspaces } from './workspaces';
+import { workspaceOne, workspaceTwo } from './workspaces';
 
 const TYPE = 'programs';
 
-let programs;
+export const testPrograms = [];
```

Comment:
Do we need to export this? Doesn't look like we're using them anywhere else besides this file.

Reply:
Same comment for `testWorkspaces` in the other `/support/api/workspaces.js` file.

Reply:
good catch.. I don't think so.. this was part of an earlier though, but didn't end up using it... better without